### PR TITLE
initial releasenotes for 15th december

### PR DESCRIPTION
**Pull request by rebert**

### CODE REVIEW

File: chsdi/static/doc/source/releasenotes/index.rst [hunk 1]
```diff
@@ -27,6 +27,10 @@ API & applications
 `API <//api3.geo.admin.ch>`__
 '''''''''''''''''''''''''''''
 - Bug fixes
+- Announcement: the layer ch.babs.kulturgueter-anhoerung has been removed from FSDI services as previously announced
+- Announcement: the layer ch.bakom.downlink3 has been removed from FSDI services as previously announced
```
Comment (marionb) [inline chsdi/static/doc/source/releasenotes/index.rst:31]: @consta-v is this correct that it is added here already? I did not do anything yet for this decommissioning as you told me I can take a look later.

Comment (mariokeusen) [reply chsdi/static/doc/source/releasenotes/index.rst:31]: @marionb I have taken this layer from prod to test for the deploy, next real update on the automat will be somewhen in summer, so we have no hurries here.

Comment (consta-v) [reply chsdi/static/doc/source/releasenotes/index.rst:31]: Correct

**Comment by faselm** [comment]
Hi, how I can see why the Python tests are failing now? I just added additional layers like in the first commit that is OK...

**Comment by b4l** [comment]
@faselm you can run `make lint` on your local instance.

**Comment by faselm** [comment]
@b4l Thanks, didn't notice the additional spaces!

**Review by ltclm — APPROVED** [review]

**Pull request merged**